drivers/scheduler/openshift/openshift.go

```diff
@@ -277,6 +580,57 @@ func getCluterInfo() string {
 	return output.(string)
 }
 
+func ackAPIRemoval(version string) error {
```

Comment:
these changes shoudn't be in your PR, rebase your branch with master and fix the conflicts

Reply:
@thiguetta It only the line number changes, don't know why it is showing in my PR

Reply:
Resolved..